appcenter-data/Data.d.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export const enum TimeToLive {
+    INFINITE = -1,
+    NO_CACHE = 0,
+    DEFAULT = -1
+}
+
+export const enum DefaultPartitions {
+    USER_DOCUMENTS = 'user',
+    APP_DOCUMENTS = 'readonly'
+}
+
+export class ReadOptions {
+    timeToLive: number | TimeToLive;
+    constructor(timeToLive: number | TimeToLive);
+}
+
+export class WriteOptions {
+    timeToLive: number | TimeToLive;
+    constructor(timeToLive: number | TimeToLive);
+}
+
+export interface DocumentWrapper {
+    deserializedValue: object;
+    eTag: string;
+    id: string;
+    isFromDeviceCache: boolean;
+    jsonValue: string;
```

Comment:
In strict mode, values are not nullable unless specified otherwise. We should add `| null` on some of the properties that can have a null value.

Reply:
Added